softoff/softoff.cpp

```diff
@@ -397,6 +413,8 @@ int SoftPowerOff::hostSoftOff(sdeventplus::Event& event)
             "RC", static_cast<int>(rc), "ERR", errno);
         return PLDM_ERROR;
     }
+    std::vector<uint8_t> requestbuffer(requestMsg.begin(), requestMsg.end());
+    pldm::utils::printBuffer(pldm::utils::Tx, requestbuffer);
```

Comment:
is this change not upstreamed ? should not we upstream ?

Reply:
nope, will create a gerrit patch for this.